src/review/forms.py

```diff
@@ -209,8 +209,12 @@ class FakeReviewerDecisionForm(FakeModelForm, ReviewerDecisionForm):
 
     def __init__(self, *args, **kwargs):
         kwargs["disable_fields"] = True
+        open_peer_review = kwargs.pop('open_peer_review')
         super().__init__(*args, **kwargs)
 
+        if not open_peer_review.value:
```

Comment:
Should be defensive against `open_peer_review` not being a setting (e.g. being `None`):

`if open_peer_review and open_peer_review.value:`